app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -21,7 +20,6 @@ module HostManagedExtensions
 
         before_save :correct_puppet_environment
 
-        scoped_search :in => :content_source, :on => :name, :complete_value => true, :rename => :content_source
```

Comment:
I think we want to keep the ability to search for thsi field, probably want to move it to ContentFacetHostExtensions